gvsbuild/projects/opus.py

```diff
@@ -13,34 +13,32 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program; if not, see <http://www.gnu.org/licenses/>.
 
-from gvsbuild.utils.base_builders import CmakeProject
+from gvsbuild.utils.base_builders import Meson
 from gvsbuild.utils.base_expanders import Tarball
 from gvsbuild.utils.base_project import project_add
 
 
 @project_add
-class Opus(Tarball, CmakeProject):
+class Opus(Tarball, Meson):
     def __init__(self):
-        CmakeProject.__init__(
+        Meson.__init__(
             self,
             "opus",
-            version="1.5.1",
+            version="1.5.2",
             repository="https://github.com/xiph/opus",
             archive_url="https://downloads.xiph.org/releases/opus/opus-{version}.tar.gz",
-            hash="b84610959b8d417b611aa12a22565e0a3732097c6389d19098d844543e340f85",
+            hash="65c1d2f78b9f2fb20082c38cbe47c951ad5839345876e46941612ee87f9a7ce1",
             dependencies=[
                 "ninja",
-                "cmake",
+                "meson",
                 "pkgconf",
             ],
         )
+        self.add_param("-Dtests=disabled")
+        self.add_param("-Ddocs=disabled")
 
     def build(self):
-        CmakeProject.build(
-            self,
-            use_ninja=True,
-            cmake_params="-DOPUS_BUILD_SHARED_LIBRARY=ON -DOPUS_BUILD_TESTING=OFF",
-        )
+        Meson.build(self)
         self.install(r"COPYING share\doc\opus")
 
         configuration = (
```

Comment:
This chunk needs to be removed as well I think. It was added in https://github.com/wingtk/gvsbuild/commit/8d5e456ceebdfb55126d0db3a6b7600657ac843a

Reply:
Thanks @pbor, good catch!